Doyeon-k-fanshawe/group2_bootcamp
# Updating Promotional Video

## What does the PR do?
* It updates the promotional video in the assets folder

## How do you check this PR?
* Pull down the branch
*  See changed video

## Screenshots 
None.
